modules/ossm-threescale-applying-external-service-entry-objects.adoc

```diff
@@ -49,6 +49,37 @@ spec:
   location: MESH_EXTERNAL
   resolution: DNS
 ----
++
+.`DestinationRule` custom resource for 3scale Hosted backend
+[source,terminal]
+----
+apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+  name: threescale-saas-backend
+spec:
+  host: su1.3scale.net
+  trafficPolicy:
+    tls:
+      mode: SIMPLE
+      sni: su1.3scale.net
+----
++
+.`DestinationRule` custom resource for 3scale Hosted system
+[source,terminal]
+----
+apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+  name: threescale-saas-system
+spec:
+  host: multitenant.3scale.net
+  trafficPolicy:
+    tls:
+      mode: SIMPLE
+      sni: multitenant.3scale.net
+----
+
```

Comment:
Line 84 - The flow here might be better if you said something like, "Use the following command to apply external ServiceEntry and related DestinationRule custom resources to your cluster." list the commands, and the after the commands list the YAML files.

Line 100 "To do this, change the location of these services in the custom resources."  these services is vague, we should be clear about which services should be moved to achieve the goal here.

Reply:
Thanks @JStickler 

@rahulanand16nov 
Can I get clarification on the following lines regarding "these services": 

`Alternatively, you can deploy an in-mesh 3scale service. To do this, change the location of these services in the CRs.`

Reply:
@dfennessy Here, we are using the URL for the 3scale SaaS version, but it doesn't have to be; users can deploy their 3scale and use links to that deployment.